_sources/quickcheck-contractmodel/0.1.3.0/revisions/1.cabal

```
@@ -0,0 +1,104 @@
+cabal-version:      2.4
+name:               quickcheck-contractmodel
+version:            0.1.3.0
+
+-- A short (one-line) description of the package.
+-- synopsis:
+
+-- A longer description of the package.
+-- description:
+
+-- A URL where users can report bugs.
+-- bug-reports:
+
+-- The license under which the package is released.
+-- license:
+author:             Maximilian Algehed
+maintainer:         maximilian.algehed@quviq.com
+
+-- A copyright notice.
+-- copyright:
+-- category:
+
+common lang
+  default-language:   Haskell2010
+  default-extensions:
+    FunctionalDependencies
+    RecordWildCards
+    ViewPatterns
+    DeriveFunctor
+    DeriveDataTypeable
+    StandaloneDeriving
+    ImportQualifiedPost
+    TupleSections
+    LambdaCase
+    PatternSynonyms
+    GADTs
+    TypeApplications
+    ScopedTypeVariables
+    TypeFamilies
+    FlexibleContexts
+    FlexibleInstances
+    MultiParamTypeClasses
+    RankNTypes
+    GeneralizedNewtypeDeriving
+    PolyKinds
+    TemplateHaskell
+    DataKinds
+    DefaultSignatures
+    ConstraintKinds
+    DeriveGeneric
+    DerivingVia
+    TypeOperators
+    OverloadedStrings
+
+  ghc-options:
+    -Wall -Wnoncanonical-monad-instances -Wunused-packages
+    -Wincomplete-uni-patterns -Wincomplete-record-updates
+    -Wredundant-constraints -Widentities -Wno-name-shadowing
+    -Wno-orphans -Wno-redundant-constraints
+
+library
+    import: lang
+    hs-source-dirs: src
+    exposed-modules:
+      Test.QuickCheck.ContractModel
+      Test.QuickCheck.ContractModel.DL
+      Test.QuickCheck.ContractModel.Internal
+      Test.QuickCheck.ContractModel.Internal.Model
+      Test.QuickCheck.ContractModel.Internal.Common
+      Test.QuickCheck.ContractModel.Internal.ChainIndex
+      Test.QuickCheck.ContractModel.Internal.Spec
+      Test.QuickCheck.ContractModel.Internal.Symbolics
+      Test.QuickCheck.ContractModel.Internal.Utils
+      Test.QuickCheck.ContractModel.ThreatModel
+      Test.QuickCheck.ContractModel.ThreatModel.Cardano.Api
+      Test.QuickCheck.ContractModel.ThreatModel.DoubleSatisfaction
+      Test.QuickCheck.ContractModel.ThreatModel.Pretty
+      Test.QuickCheck.ContractModel.ThreatModel.TxModifier
+    -- General dependencies
+    build-depends:
+      base >=4.7 && <5,
+      bytestring ^>= 0.10.12,
+      containers ^>= 0.6.5.1,
+      lens ^>= 5.0 || ^>= 5.1 || ^>= 5.2,
+      mmorph ^>= 1.2,
+      mtl ^>= 2.2.2,
+      pretty ^>= 1.1.3.6,
+      time ^>= 1.9.3,
+    -- QuickCheck dependencies
+      QuickCheck ^>= 2.14,
+      quickcheck-dynamic >= 3.0.2,
```

```suggestion
      quickcheck-dynamic ^>= 3.0.2,
```
did you miss the edit?

---

Yes, already pushed it.

---

That was it. :tada:

---

Checked and it's already upstream, so no need to do that.